.eslintrc.cjs

```diff
@@ -29,20 +30,47 @@ module.exports = {
     "react/jsx-props-no-spreading": 0,
     "react/jsx-wrap-multilines": 0,
     "react/prefer-stateless-function": 0,
-    "import/order": [
+    "import/order": 0,
+    "sort-imports": 0,
+    // perfectionist
+    "perfectionist/sort-imports": [
       "error",
       {
-        alphabetize: { order: "asc" },
-        groups: ["builtin", "external", "internal", "parent", "sibling", "index"],
-        pathGroups: [
-          {
-            // CSS should be loaded last so that it can override components in common folder
-            pattern: "./*.module.css",
-            group: "sibling",
-            position: "after",
-          },
+        groups: [
+          ["builtin", "external", "type"],
+          ["internal", "internal-type"],
+          ["parent-type", "parent"],
+          ["index-type", "index"],
+          ["sibling-type", "sibling"],
+          "side-effect",
+          "style",
+          "object",
+          "unknown",
+        ],
+        "internal-pattern": [
+          "~/**",
+          "@/**",
+          "__generated__/**",
+          "constants/**",
+          "components/**",
+          "e2e/**",
+          "helpers/**",
+          "hooks/**",
+          "pages/**",
+          "styles/**",
```

Comment:
can remove these, not relevant. The `internal-pattern` should contain any aliases/paths/etc that we use. Check tsconfigs, etc for this